src/main/java/org/sefglobal/scholarx/service/IntrospectionService.java

```diff
@@ -194,7 +194,7 @@ public Mentee confirmMentor(long mentorId, long profileId)
             throw new ResourceNotFoundException(msg);
         }
 
-        Optional<Mentee> optionalMentee = menteeRepository.findByProfileIdAndMentorId(profileId, mentorId);
+        Optional<Mentee> optionalMentee = menteeRepository.findByProfileIdAndAppliedMentorId(profileId, mentorId);
         if (!optionalMentee.isPresent()) {
             String msg = "Error, User with id: " + profileId + " haven't applied for " +
                          "mentor by id: " + mentorId + ".";
```

Comment:
do we still need this confirm mentor function??

Reply:
No, we don't. I created a separate issue for removing it. @Gimhan-minion is working on that. So I just updated the method to pass the tests.

Reply:
Alright